### PR TITLE
8281711: Cherry-pick WebKit 613.1 stabilization fixes

### PR DESCRIPTION
**Pull request by arapte**
Include additional stabilization fixes of WebKit 613.1
Sanity testing did not show any concerns.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8281711](https://bugs.openjdk.java.net/browse/JDK-8281711): Cherry-pick WebKit 613.1 stabilization fixes

### Reviewers
 * [Johan Vos](https://openjdk.java.net/census#jvos) (@johanvos - **Reviewer**)
 * [Kevin Rushforth](https://openjdk.java.net/census#kcr) (@kevinrushforth - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jfx pull/733/head:pull/733` \
`$ git checkout pull/733`

Update a local copy of the PR: \
`$ git checkout pull/733` \
`$ git pull https://git.openjdk.java.net/jfx pull/733/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 733`

View PR using the GUI difftool: \
`$ git pr show -t 733`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jfx/pull/733.diff">https://git.openjdk.java.net/jfx/pull/733.diff</a>

</details>

### CODE REVIEW

File: modules/javafx.web/src/main/native/Source/JavaScriptCore/API/JSRetainPtr.h [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * are met:
  *
  * 1.  Redistributions of source code must retain the above copyright
- *     notice, this list of conditions and the following disclaimer.
+q *     notice, this list of conditions and the following disclaimer.
```
Comment (danielpeintner) [inline modules/javafx.web/src/main/native/Source/JavaScriptCore/API/JSRetainPtr.h:9]: seems to be unintentional

Comment (arapte) [reply modules/javafx.web/src/main/native/Source/JavaScriptCore/API/JSRetainPtr.h]: Thank you, It is corrected now.

**Comment by kevinrushforth** [comment]
So far all my testing looks good, but I'll finish tomorrow.

**Comment by johanvos** [comment]
Looks good on Linux.

**Review by johanvos — APPROVED** [review]

**Review by kevinrushforth — APPROVED** [review]

**Pull request closed without merging**